### PR TITLE
fix(frontend): make OAuth request client-side and include credentials in it

### PR DESCRIPTION
**Pull request by Clement-Z4RM**
- The OAuth API request (actually only for google, `/oauth/google`) includes now crdentials (cf. https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials)
- The request is made on the client-side instead of server side since it needs things like client-session, only accessible on client

- @ElouanR, I remaned the schemas in the documentation to avoid some characters like `<` or `|` which are forbidden in file names/paths on Windows

### CODE REVIEW

**Review by ElouanR — DISMISSED** [review]

File: frontend/src/lib/components/dashboard/area/AreaForm/AreaForm.svelte [hunk 1]
```diff
@@ -11,7 +11,7 @@
         description: string;
     };
 
-    // TODO: replace with backend data
+    // TODO: replace with backend data or at least move it to common
     const FIELDS: Record<string, Field[]> = {
         "youtube.on_liked_video": [
             { name: "id", displayName: "ID", type: "text", description: "The identifiant of the video" },
```
Comment (Ximaz) [inline frontend/src/lib/components/dashboard/area/AreaForm/AreaForm.svelte:17]: `identifier` instead of `identifiant`.

**Comment by Ximaz** [comment]
WHat's up with the mobile CI ?

**Comment by Clement-Z4RM** [comment]
> What's up with the mobile CI ?

Because mobile can't import `common` from `/common/`, @capuchegianni copied it in `/mobile/common/`.
But `common` imports things from `backend`, and *ESLint* don't find them.
I didn't quite understand the problem, so I had made a `npm ci` in `/mobile/common/` in mobile CI, but this wasn't the problem.
I'll fix it with monorepo or whatever solution to use real `common` (`/common/`) in mobile, so for now let's let it as is.

File: frontend/src/lib/components/dashboard/area/AreaForm/AreaForm.svelte (hunk 1 above)
Comment (Clement-Z4RM) [reply frontend/src/lib/components/dashboard/area/AreaForm/AreaForm.svelte]: Fixed in 4dedd120

**Review by Ximaz — APPROVED** [review]
lgtm

**Pull request merged**